geom/geom/inc/TGeoVolume.h

```diff
@@ -204,6 +205,7 @@ class TGeoVolume : public TNamed, public TGeoAtt, public TAttLine, public TAttFi
    void RemoveNode(TGeoNode *node);
    TGeoNode *ReplaceNode(TGeoNode *nodeorig, TGeoShape *newshape = nullptr, TGeoMatrix *newpos = nullptr,
                          TGeoMedium *newmed = nullptr);
+   void ResetTransparency(Char_t transparency = -1);
```

Comment:
```suggestion
   void ResetTransparency(Char_t transparency = -1); // *MENU*
```
Otherwise the method does not appear in the context menu (right-click) of the volume objects.